README.md

```diff
@@ -53,7 +55,12 @@ The script can run on GitBash, Windows Subsystem for Linux and Cygwin terminals.
 
 If using Cygwin, then please ensure `curl` and `unzip` packages are installed. You will need to run the Cygwin setup again.
 
-Note: **PowerShell is Unsupported**
+Note: **PowerShell is Unsupported**.
+
+## Firewall Requirements
+
+1. Ensure HTTPS (**port: 443**) and SSH (**port: 22**) is allowed.
+2. Ensure **outbound** access is allowed.
```

Comment:
We may also need to give 6443 port for CLI access. Better point to the [ocp-power-automation doc link](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/docs/automation_host_prereqs.md#configure-your-firewall) to prevent duplication?

Reply:
443 and 22 is more from ingress pov. Although I'm not exactly sure if 443 is needed for ingress or not. Will need to recheck.
All outbound will ensure access oc can access 6443 and any other like yum repos etc